### PR TITLE
 Resolved errors and warnings related to javadoc

### PR DESCRIPTION
**Pull request by emmetsap**
Resolving errors and warnings including completion of Javadoc comments

### CODE REVIEW

File: services/distribution/src/main/java/app/coronawarn/server/services/distribution/objectstore/S3Publisher.java [hunk 1]
```diff
@@ -69,7 +69,6 @@ public S3Publisher(ObjectStoreAccess objectStoreAccess, FailedObjectStoreOperati
    * be controlled through the configuration parameter <code>DistributionServiceConfig.forceUpdateKeyFiles</code>
    *
    * @param root The path of the directory that shall be published.
-   * @see Github issue #650
```
Comment (hilmarf) [inline services/distribution/src/main/java/app/coronawarn/server/services/distribution/objectstore/S3Publisher.java:72]: @emmetsap - OK, `@see` doesn't work... but we should keep the information/reference to the particular github issue somewhere else

File: services/download/src/main/java/app/coronawarn/server/services/download/FederationBatchProcessor.java [hunk 2]
```diff
@@ -44,10 +44,10 @@
...
   private final DownloadServiceConfig config;
   private final ValidFederationKeyFilter validFederationKeyFilter;
-  
+
   // This is a potential memory-leak if there are very many batches
-  // This is an intentional decision: 
-  // We'd rather run into a memory-leak if there are too many batches 
+  // This is an intentional decision:
+  // We'd rather run into a memory-leak if there are too many batches
   // than run into an endless loop if a batch-tag repeats
```
Comment (hilmarf) [inline services/download/src/main/java/app/coronawarn/server/services/download/FederationBatchProcessor.java:51]: please turn this into real JavaDoc comment

**Comment by hilmarf** [comment]
@emmetsap can you please resolve the conflicts?

**Review by hilmarf — DISMISSED** [review]

File: services/distribution/src/main/java/app/coronawarn/server/services/distribution/objectstore/S3Publisher.java (hunk 1 above)
Comment (emmetsap) [reply services/distribution/src/main/java/app/coronawarn/server/services/distribution/objectstore/S3Publisher.java:72]: I've now moved the reference below the description

File: services/download/src/main/java/app/coronawarn/server/services/download/FederationBatchProcessor.java (hunk 2 above)
Comment (emmetsap) [reply services/download/src/main/java/app/coronawarn/server/services/download/FederationBatchProcessor.java]: I've made this change

**Review by hilmarf — APPROVED** [review]

**Review by FelixRottler — APPROVED** [review]

**Pull request merged**